src/main/java/hello/HelloController.java

```diff
@@ -36,4 +36,16 @@ public String index() {
     public String getUser() {
         return "John Doe";
     }
+
+    public Object clone() {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Remove this "clone" implementation; use a copy constructor or copy factory instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.api.cicd.siriuscloudservices.com/coding_rules#rule_key=squid%3AS2975)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use super.clone() to create and seed the cloned instance to be returned. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.api.cicd.siriuscloudservices.com/coding_rules#rule_key=squid%3AS1182)